src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <section>
+        <div className="calcDiv">
+          <button type="button" className="btn" value="AC">AC</button>
```

Comment:
- [ ] maybe you forget the screen of the calculator but you should implement a screen of the calculator as in the design 
- Yours 
<img width="869" alt="Screen Shot 2022-08-04 at 16 00 03" src="https://user-images.githubusercontent.com/61790587/182880656-24c6a3b5-bfcc-44bb-a1f4-ae710316bbdb.png">
- the screen you forgot. 
<img width="388" alt="Screen Shot 2022-08-04 at 16 00 36" src="https://user-images.githubusercontent.com/61790587/182880729-f1c97054-d238-4c65-81af-daf24af408a8.png">